app/models/agents/data_output_agent.rb

```diff
@@ -128,12 +169,13 @@ def receive_web_request(params, method, format)
             <?xml version="1.0" encoding="UTF-8" ?>
             <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
             <channel>
-             <atom:link href="#{feed_link.encode(:xml => :text)}/users/#{user.id}/web_requests/#{id || '<id>'}/#{params['secret']}.xml" rel="self" type="application/rss+xml" />
-             <title>#{feed_title.encode(:xml => :text)}</title>
-             <description>#{feed_description.encode(:xml => :text)}</description>
-             <link>#{feed_link.encode(:xml => :text)}</link>
-             <lastBuildDate>#{Time.now.rfc2822.to_s.encode(:xml => :text)}</lastBuildDate>
-             <pubDate>#{Time.now.rfc2822.to_s.encode(:xml => :text)}</pubDate>
+             <atom:link href=#{feed_url(secret: params['secret'], format: :xml).encode(xml: :attr)} rel="self" type="application/rss+xml" />
```

Comment:
Yes, it returns a quoted string. https://github.com/ruby/ruby/blob/ruby_1_9_3/transcode.c#L2868-L2869